src/components/DisplayConfirmPlayer.tsx

```diff
@@ -0,0 +1,28 @@
+import Container from "@mui/material/Container";
+import Button from "@mui/material/Button";
```

Comment:
おそらくこれでひとまとめにできるはずです
```suggestion
import { Button, Container } from "@mui/material";
```